### PR TITLE
Operator Api | Add driver set password endpoint

### PR DESCRIPTION
**Pull request by tom-ioki**
This PR will introduce the `driver_set_password` endpoint.

### CODE REVIEW

File: spec/ioki/operator_api_spec.rb [hunk 1]
```diff
@@ -260,15 +260,16 @@
     end
   end
 
-  describe '#drivers_set_password(product_id, driver_id, password)' do
+  describe '#update_set_password(product_id, driver_id, ...)' do
```
Comment (phylor) [inline spec/ioki/operator_api_spec.rb:263]:
```suggestion
  describe '#update_driver_password(product_id, driver_id, ...)' do
```

Should this have the same name as the method?

**Review by phylor — APPROVED** [review]
I was wondering why you are moving `set_password` to a new `Endpoints::Update`. As far as I understand, `Endpoints.custom_endpoints` does not send a body with the requests. So we have to explicitly use `Endpoints::Update` if we want to serialize our model. Is that correct?

If so, we could add a comment to `custom_endpoints` to make that clear.

**Comment by tom-ioki** [comment]
> I was wondering why you are moving `set_password` to a new `Endpoints::Update`. As far as I understand, `Endpoints.custom_endpoints` does not send a body with the requests. So we have to explicitly use `Endpoints::Update` if we want to serialize our model. Is that correct?
> 
> If so, we could add a comment to `custom_endpoints` to make that clear.

Yes, that is correct! I was not able to send in the password with the custom endpoint, hence I introduced the Update endpoint.

Do you mean commenting in the `CustomEndpoints` class, or where should we put that comment?

File: spec/ioki/operator_api_spec.rb (hunk 1 above)
Comment (tom-ioki) [reply spec/ioki/operator_api_spec.rb:263]: yes!

**Pull request merged**